spec/lib/telephony/notification_sender_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require 'nokogiri'
+require 'rails_helper'
+
+RSpec.describe Telephony::NotificationSender do
+  before do
+    Telephony::Test::Message.clear_messages
+    Telephony::Test::Call.clear_calls
+  end
+
+  context 'with the test adapter' do
+    subject do
+      described_class.new(
+        message: message,
+        to: to,
+        expiration: expiration,
+        channel: channel,
+        domain: domain,
+        country_code: country_code,
+        extra_metadata: { phone_fingerprint: 'abc123' },
+      )
+    end
+
+    let(:to) { '+1 (202) 262-1234' }
+    let(:expiration) { 5 }
```

Comment:
seeing this value for `expiration` made me wonder what measurement of time this represents

Reply:
@svalexander This is a copy of test the class stated in the card.

Reply:
Not valid anymore.